### PR TITLE
RN 36611

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Below you can find an overview of the different fields in the CcaGateway window and how these can be used.
+
+> [!CAUTION]
+> The majority of these fields are only intended to be used by Skyline DevOps Engineers. Do not make any changes in this window unless you have been explicitly asked to do so by a Skyline DevOps Engineer.
+
+![CcaGateway window](~/user-guide/images/CcaGatewayWindow.png)
+
+1. The *Request Info* box shows the status of the last request done using this window. This can be used to see potential errors in communication.
+
+1. The *DCP Connect Endpoint* box shows the current connection endpoint used to make the cloud connection.
```
Comment (JordyGit) [inline user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:32]:
```suggestion
1. The *DCP Connect Endpoint* box shows the current connection endpoint used to make and maintain the cloud connection.
```

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+1. [Connect to the DMA using the SLNetClientTest tool](xref:Connecting_to_a_DMA_with_the_SLNetClientTest_tool).
+
+1. Go to *Advanced* > *CcaGateway*.
+
+When the DataMiner Agent is **offline**:
+
+- Go to *Offline Tools* > *CcaGateway (offline)*.
+
+## CcaGateway window UI
```
Comment (JordyGit) [inline user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:21]: Note that the window has been adjusted over time.
All versions have the Connect & Tunnel Endpoint (name in UI could vary but they are the same fields)
Some older versions had **no or multiple other endpoint fields** that now all have been merged into one: DCP API Endpoint

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+1. The *Set* button should only be used by Skyline DevOps Engineers. It can be used to change the *DCP Connect Endpoint*, *DCP Tunnel Endpoint*, and *DCP API Endpoint* value to the value that is currently filled in in the corresponding boxes.
+
+1. The *Get Global State* button retrieves the state of the dataminer.services connection and displays it in the box on the right. This can be useful to retrieve the cloud identity of your system, which may be needed for support.
+
+1. The *Register DMS* button behaves the same way as the *Connect* button in DataMiner Cube. We strongly recommend using DataMiner Cube instead. See [Connecting to dataminer.services](xref:Connect_to_dataminer_services).
+
+1. The *Unregister DMS* button removes the connection to dataminer.services and thereby disconnects your DMS from the cloud. **Do not use this option unless you are specifically asked by a Skyline DevOps engineer. This action cannot be undone.**
+
+1. The *Renew Cloud Session* button refreshes the cloud session. However, we strongly recommend using the *Renew* button on the *Cloud* page in System Center for this instead.
```
Comment (JordyGit) [inline user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:48]: (will only be visible if cloud connection is not valid)

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md (hunk 2 above)
Comment (MariekeGO) [reply user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:21]: I've added the DataMiner version below the screenshot to deal with possible UI differences in different versions.

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+1. The *Validate Cloud Session* button checks the status of the current dataminer.services connection. You can also see this in Cube, where it is visualized by either a green checkmark or a red x.
+
+1. The *Link Accounts Session* button is used to link a DataMiner account to a dataminer.services account. However, this should only be used by Skyline DevOps Engineers. We recommend using the procedure detailed under [Linking your DataMiner and DCP account](xref:Linking_your_DataMiner_and_DCP_account).
```
Comment (JordyGit) [inline user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:53]: IMO it wouldn't harm if they used it

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+1. The *Get Node Info* button retrieves information on the CloudGateway installation and displays it in the table below.
```
Comment (JordyGit) [inline user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:55]: I think this part was deprecated (also the info is not correct anymore if i check it now (showing i'm not cloud connected for example), we maybe should remove it)

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md (hunk 4 above)
Comment (MariekeGO) [reply user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:53]: OK to change to this?

```suggestion
1. The *Link Accounts Session* button is used to link a DataMiner account to a dataminer.services account. However, we recommend using the procedure detailed under [Linking your DataMiner and DCP account](xref:Linking_your_DataMiner_and_DCP_account) instead.
```

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md (hunk 5 above)
Comment (MariekeGO) [reply user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:55]: OK to change to this?

```suggestion
1. The *Get Node Info* button retrieves information on the CloudGateway installation. This button is now **deprecated**.
```

**Review by LouisSkyline — APPROVED** [review]

File: user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md (hunk 4 above)
Comment (JordyGit) [reply user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.md:53]: yes

**Pull request merged**